spec/remote_helper.rb

```diff
@@ -0,0 +1,2 @@
+require 'dotenv'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.